### DISCUSSION TITLE
Ignore NaN in position with Deeplabcut?

### DISCUSSION THREAD

**Discussion by AnthonyC-S** [bonsai-rx/bonsai#1330] [Q&A] [answered]
I'm tracking a position with Deeplabcut GetBodyPart followed by ContainsPoint for a ROI. Any suggestions or easy ways to ignore NaN's in the position and use the last known position instead?

**Comment by bruno-f-cruz** [comment, marked as answer]
!!!Ignore my previous answer.

You can test if the incoming value is a NaN and drop it using a `Condition`:
![image](https://user-images.githubusercontent.com/7049351/233575853-e998a4c0-704b-4d21-9d49-830e4976a282.png)

`ExpressionTransform` with `Double.IsNaN(it)` as the expression.

**Reply by AnthonyC-S** [reply]
Thank you!

**Comment by edmundchong** [comment]
I use Double.IsNan(it)==False in an `ExpressionCondition` for brevity :)
